VPLAY-9707:fix multi-character character constant warnings

srikanthreddybijjam-comcast opened the pull request:
Reason for change:warning due to multiple char conversion to uint32_t 
Test Procedure: Refer VPLAY-9707 for clearing warnings due to type conversion errors. 
Priority: P2

Signed-off-by: Srikanth2000<srikanthreddybijjam.2000@gmail.com>

srikanthreddybijjam-comcast committed:
VPLAY-9707:fix multi-character character constant warnings

Reason for change:warning due to multiple char conversion to uint32_t
Test Procedure: Refer VPLAY-9707 for clearing warnings due to type conversion errors.
Priority: P2

Signed-off-by: Srikanth2000<srikanthreddybijjam.2000@gmail.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=122) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=122) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: Vinish100<br/>:white_check_mark: srikanthreddybijjam-comcast<br/>:white_check_mark: jfagunde<br/>:x: Srikanth<br/><hr/>**Srikanth** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=122) it.</sub>

Vinish100 committed:
Revert "VPLAY-9299: Address 200ms tune delay when using enableMediaProcessor as true"

This reverts commit edac63eb4c7ddfafef368ffceb1c723312e5f345.
This reverts commit 056e0e50561b845dd1bbcc34967a59280218b061.

jfagunde committed:
VPLAY-9675 Use StreamLock to protect StreamAbstraction obj in detach() (#75)

Reason for change: Fix a crash when playing VoD

Test Procedure: Reproduce sequence in Jira description

Risks: Medium

Change-Id: I331c9c4805d9bb2227ee60cb9e386f388c3cf5d6

Signed-off-by: Jose Fagundez <jfagunde@synamedia.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

jfagunde committed:
VPLAY-9794 [ENT-4082] Use chunk mode flag for TotalInjectedDuration calc (#108)

Reason for change: Calculate the right total injected duration

Test Procedure: Verify log line from GetBufferStatus

Risks: Low

Signed-off-by: Jose Fagundez <jfagunde@synamedia.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

Vinish100 commented on test/utests/tests/DrmOcdm/DrmHelperTests.cpp:
```diff
@@ -391,6 +391,14 @@ TEST_F(DrmHelperTests, TestCreatePlayReadyHelperNegative)
 	}
 }
 
+static uint32_t Decimalvalue(const char* s)
```
ConvertToDecimal

Commit:
VPLAY-9707:Video Freeze seen during CDAI ADS

Reason for change: Modified Multi char Warnings
Test Procedure:  Refer VPLAY-9707 for clearing warnings due to type conversion errors.
Priority: P2

Signed-off-by: Srikanth2000<srikanthreddybijjam.2000@gmail.com>
